src/app/containers/ArticleMetadata/__snapshots__/index.test.jsx.snap

```diff
@@ -263,7 +263,7 @@ exports[`should match snapshot for Persian News & UK origin 1`] = `
       rel="amphtml"
     />
     <link
-      href="/persian/images/icons/icon-192x192.png"
```

Comment:
@thekp is this expected? have you checked that the icons can be found at `http://localhost.bbc.com:7080/persian/images/icons/icon-192x192.png` ?

Reply:
They wouldn't be found at `http://localhost.bbc.com:7080/` however on a live page we would expect the `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` origin the exist. 

I've set it to localhost here: https://github.com/bbc/simorgh/pull/4181/files#diff-77cca5a3ad8587ce98baf1f2d6534917R22

to match our local environment config (./enConfig/local.env)

Reply:
The snapshot to be less deceptive, should I make a change to remove the `http://localhost.bbc.com:7080/` from it.

I would do this by removing (SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN) it from the jest config.